src/ragas/metrics/_simple_criteria.py

```diff
@@ -124,7 +124,7 @@ def __init__(
         self.multi_turn_prompt = multi_turn_prompt or MultiTurnSimpleCriteriaPrompt()
 
         # update the instruction for the prompts with the definition
-        instruction = f"Evaluate the Input based on the criterial defined. Give a score between 0 and 5.\nCriteria Definition: {self._definition}"
+        instruction = f"Evaluate the input based on the criteria defined. Give a score between 0 and 5.\nCriteria Definition: {self._definition}"
```

Comment:
Here it's better to keep the instruction and the act of appending instruction and criteria different. For here, move the instruction to SingleTurnSimpleCriteriaPrompt.